library/src/main/java/org/akop/ararat/view/CrosswordView.kt

```diff
@@ -623,11 +634,10 @@ class CrosswordView(context: Context, attrs: AttributeSet?) :
                     handled = true
                 }
                 KeyEvent.KEYCODE_ENTER -> {
-                    if (MARKER_SOLVED and markerDisplayMode != 0) {
-                        if (isSelectedCellSolved() == true) {
+                    if ((MARKER_SOLVED and markerDisplayMode != 0) && isSelectedCellSolved() == true
```

Comment:
Logic is not the same. Before if `isSelectedCellSolved() != true` nothing happens, now it goes to `else` leg

Reply:
No. It is else for when